examples/sgp4Prop/unit_test/op_testrunner.c

```diff
@@ -35,8 +35,11 @@ int main(void)
 {
     OP_App_Test_AddTestCases();
     return UtTest_Run();
-} /* End of main() */
+} 
+
+/* End of main() */
```

Comment:
Can you revert these changes to these unrelated files?

Reply:
Reverted one file that I added a couple extra blank lines, the other file I re-commented a main function that was supposed to be commented out, may have been left out from past pull request.